roles/confluent.common/molecule/custom-vars/files/custom.repo

```diff
@@ -0,0 +1,13 @@
+[Confluent.dist]
+baseurl = https://packages.confluent.io/rpm/5.3/7
```

Comment:
Isn't this repo using 5.4?

Reply:
the test was just to confirm a dummy .repo file made it in, but yes